openedx/features/course_experience/api/v1/tests/test_views.py

```diff
@@ -37,20 +36,11 @@ def test_reset_deadlines(self):
         response = self.client.post(reverse('course-experience-reset-course-deadlines'), {'course': self.course.id})
         assert response.status_code == 400
         assert enrollment.schedule == Schedule.objects.get(id=enrollment.schedule.id)
-        self.assert_no_events_were_emitted()
```

Comment:
Could you please tell me why you are doing this? The event is still emitted, but just in utils and not in view directly.

Reply:
Yeah, I saw that you're moving it to a separate file. 
Ok, next time you open the PR in the upstream, please add a comment explaining this)